src/database/utilities/createRecord.js

```diff
@@ -566,7 +566,7 @@ const createCustomerRefundLine = (database, customerCredit, transactionBatch) =>
  * @param   {Name}         customer  Customer associated with invoice.
  * @return  {Transaction}
  */
-const createCustomerInvoice = (database, customer, user, mode = 'store') => {
+const createCustomerInvoice = (database, customer, user, mode = 'store', customData = null) => {
```

Comment:
`JSON.stringify(null) === 'null'` - maybe not what we want? think just `''` when empty?

Reply:
Eep, thanks

Reply:
Did a bit more testing, think it's better to set the record value to `null` if not provided (for consistency with `parseJSONData` in `incomingSyncUtils`) so transactions without `customData` in mobile are consistent with those synced from Desktop 🤣  
(prev impl had records from Desktop with `null` and those from Mobile with `''` - oops)

Soz bit late response as well, too much fun testing different JSON schemas out 😹 
25641e3

Reply:
Ah, good points!